### ISSUE TITLE
Incorrect description for `count` input/output type

### ISSUE THREAD

**Issue by leeping-ng** [aimakerspace/PeekingDuck#542]
`count` which is currently an output from `dabble.bbox_count` should just be an `int` type, but the description in RTD is incorrect: 
```
count (List[int]): A list of integers representing the count of a pre-selected object (for example, ‘people’) detected in each frame through bboxes.
```

**Comment by leeping-ng** [comment]
It could have been the initial intention to present it as a list to show different counts for different classes, but with the update to `draw.legend`, it was changed to count across all bboxes for simplicity. Going forward with new nodes coming out in v1.2 which will make use of `count`, suggest retaining it as an `int` type and updating the docs.

Here, it shows count is a single int of 3 even with 2 classes, "dog" and "person".
```
'bboxes': array([[0.31895882, 0.6725785 , 0.5876959 , 0.97450596],
        [0.44654298, 0.07115207, 0.7607977 , 0.7644112 ],
        [0.22748163, 0.04981984, 0.50799876, 0.6592544 ]], dtype=float32),
 'bbox_labels': array(['dog', 'person', 'person'], dtype='<U6'),
 'bbox_scores': array([0.72365254, 0.63344693, 0.39939374], dtype=float32),
 'count': 3}
```

**Comment by leeping-ng** [comment]
This has been updated in the pull request for the CSRnet node. Once it is merged into `dev`, this issue will be closed.
